operations/payments.md

```diff
@@ -190,3 +190,272 @@ Adds a new alternative payment to a specified customer.
+                    ]
+                }
+            },
+            "Notes": null,
+            "SettlementId": null,
+            "ConsumedUtc": "2023-03-02T12:12:35Z",
+            "ClosedUtc": "2023-03-02T12:12:35Z",
+            "ChargedUtc": "2023-03-02T12:12:32Z",
+            "CreatedUtc": "2023-03-02T12:12:32Z",
+            "UpdatedUtc": "2023-03-02T12:12:37Z",
+            "AccountingState": "Closed",
+            "State": "Charged",
+            "Identifier": "",
+            "PaymentType": "Cash",
+            "DataDiscriminator": null,
+            "Data": null
+        }
+    ],
+    "Cursor": "d98c9611-0006-4691-a835-af2e00b170c4"
+}
+```
+
+#### Payment
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the payment. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the payment belongs to. |
+| `BillId` | string | optional | Unique identifier of the [Bill](bills.md#bill) the payment is assigned to. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](accountingcategories.md#accounting-category) the payment belongs to. |
+| `Amount` | [Amount value](#amount-value) | required | Payment's amount, negative amount represents either rebate or a payment. |
+| `OriginalAmount` | [Amount value](#amount-value) | required | Payment's original amount, negative amount represents either rebate or a payment. Contains the earliest known value in conversion chain. |
+| `Notes` | string | optional | Additional notes. |
+| `SettlementId` | string | optional | Identifier of the settled payment from the external system (ApplePay/GooglePay). | 
+| `ConsumedUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `ClosedUtc` | string | optional | Date and time of the payment bill closure in UTC timezone in ISO 8601 format. |
+| `ChargedUtc` | string | required | Charged date and time of the payment in UTC timezone in ISO 8601 format. |
+| `CreatedUtc` | string | required | Creation date and time of the payment created in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the payment in UTC timezone in ISO 8601 format. |
+| `AccountingState` | string [Accounting item state](#accounting-item-state) | required | Accounting state of the payment. |
+| `State` | string [Payment state](#payment-state) | required | Payment state of the payment. |
+| `Identifier` | string | optional | Additional unique identifier of the payment dependent on the payment type. |
+| `PaymentType` | string [Payment type](#payment-type) | required | Payment state of the payment. |
+| `DataDiscriminator` | object [Payment item data](#payment-data-discriminator) | required | Discriminator pointing to the field in the [Data](#payment-data-discriminator) field that contains additional information. |
```

Comment:
Is the definition of 'Payment data discriminator' missing?

Reply:
It's not, it's at the bottom or you mean something specific?